src/runtime_src/core/pcie/linux/scan.cpp

```diff
@@ -605,9 +606,43 @@ std::shared_ptr<pcidev::pci_device> pcidev::pci_device::lookup_peer_dev()
     return udev;
 }
 
+std::shared_ptr<pcidev::pci_device> pcidev::pci_device::lookup_parent_dev()
```

Comment:
pci_device is only for mgmt or user functions. i don't think we want it to work for pcie bridge.